site/site.go

```diff
@@ -439,12 +442,18 @@ func ExtractOrReadBinFS(dest string, siteFS fs.FS) (http.FileSystem, error) {
 		return nil, err
 	}
 
-	n, err := extractBin(dest, archive)
+	ok, err := verifyBinSha265IsCurrent(dest, siteFS)
```

Comment:
What about dev builds? Should we allow potentially serving the unverified binaries in dev to aid in debugging agent stuff without having to compile both slim and fat binaries?